MIGRATING.md

```diff
@@ -1,5 +1,9 @@
 ## Migration Guides
 
+### Migrating from versions < 13.1.0
+ * The SDK now supports PaymentIntents with `STPPaymentIntent`, which use `STPRedirectContext` in the same way that `STPSource` does
+   * `STPRedirectContextCompletionBlock` has been renamed to `STPRedirectContextSourceCompletionBlock`. It has the same signature, and Xcode should offer a deprecation warning & fix-it to help you migrate.
```

Comment:
are the two extra spaces here intentional? (can't recall if markdown ignores them or not)

Reply:
This *was* intentional, it's intended to be a sub-bullet.

Reply:
👍